drip/urls.py

```diff
@@ -0,0 +1,15 @@
+from django.conf import settings
+from django.urls import re_path
+
+from drip.views import UnsubscribeDripView
+
+urlpatterns = []
+
+if getattr(settings, "DRIP_UNSUBSCRIBE_USERS", False):
+    urlpatterns += [
+        re_path(
+            r"^drip/(?P<drip_uidb64>\w+)/(?P<uidb64>\w+)/(?P<token>[\w-]+)/$",
```

Comment:
just a suggestion:
you can use `slug` here as in this [example](https://docs.djangoproject.com/en/4.1/topics/http/urls/#example)
ignore this comment if you want